client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -608,8 +477,8 @@ export default {
         .map(el => el.name)
     },
 
-    handleChange (filterCategoryName, isSelected) {
-      this.filterManager.handleChange(filterCategoryName, isSelected)
+    handleFilterReset () {
+      this.$refs.searchField.handleReset()
```

Comment:
Can't we name the method `resetSearch`? Otherwise it sounds like filters are being reset